packages/fx-core/src/core/index.ts

```diff
@@ -726,7 +733,14 @@ export class FxCore implements v3.ICore {
   ): Promise<Result<Void, FxError>> {
     currentStage = Stage.provision;
     inputs.stage = Stage.provision;
-    if (!ctx || !ctx.solutionV2 || !ctx.contextV2 || !ctx.envInfoV2) {
+    if (!ctx || !ctx.projectSettings) {
```

Comment:
how about simplify to `!ctx?.projectSettings`

Reply:
OK